### DISCUSSION TITLE
Synchronize business shared folders with the same name on separate instances

### DISCUSSION THREAD

**Discussion by jrpelegrina** [abraunegg/onedrive#2041] [General]
Hello, I am testing the following use case: I have 2 shared folders with the same name, each one shared by a user. Namely:
-Shared Folder: Test; Shared by:user 1
-Shared Folder: Test; Shared by: user 2
The idea is to sync each of these folders with their own separate settings.

How would each business_shared_folders file have to be completed to indicate that the folder to be synchronized is the one shared by a certain user? I mean in the configuration file bussiness_shared_folders for the user1 Test folder as it is indicated that the folder to synchronize is that of this user1 and in the configuration file bussiness_shared_folders for the user2 Test folder as it is indicated that the folder to synchronize is that of this user2?

All the best

**Comment by abraunegg** [comment]
@jrpelegrina 
> I am testing the following use case: I have 2 shared folders with the same name, each one shared by a user.

This is a limitation of the client at present.

The *only* option you have, is to get 'user 1' or 'user 2' to rename & reshare one of those folders.

Feature Request https://github.com/abraunegg/onedrive/issues/1251 is designed to cater for this requirement, however, this requires an entire re-write of the OneDrive Business Shared Folders function, and, totally change the internal mechanics / how this is stored and recorded locally.

The goal here, when re-writing this feature, would be to align how the Windows client handles those shared folders, so that the usability is replicated.

There is no short term work around or fix.

**Reply by jrpelegrina** [reply]
OK
Thank you very much for the quick answer
